chain/network/src/routing.rs

```diff
@@ -769,23 +769,23 @@ impl RoutingTable {
         }
         debug!(target: "network", "try_save_edges: We are going to remove {} peers", to_save.len());
 
-        let component_nonce = self.component_nonce;
+        let old_component_nonce = self.component_nonce;
         self.component_nonce += 1;
 
         let mut update = self.store.store_update();
-        let _ = update.set_ser(ColLastComponentNonce, &[], &component_nonce);
+        let _ = update.set_ser(ColLastComponentNonce, &[], &self.component_nonce);
```

Comment:
It seems that we are not actually storing last component nonce in the database, since that would be `old_component_nonce`. Should we rename the column?

Reply:
That's a good observation. Please take a look at https://github.com/near/nearcore/pull/5140/files and https://github.com/near/nearcore/pull/5089
I renamed this variable there to have the proper meaning.